var/spack/repos/builtin/packages/salmon/package.py

```diff
@@ -85,6 +85,13 @@ class Salmon(CMakePackage):
             when="@{0}".format(ver),
         )
 
+    patch(
+        "https://github.com/COMBINE-lab/salmon/commit/ffb2a11.patch?full_index=1",
+        sha256="5ed3512bae665c1d72002911ab9ee6d213f10df63019ebd9e8e0ecde03823a73",
+        when="%gcc@13:",
```

Comment:
If they've got it merged in such that it's likely it'll be in a future release, could you add a version restriction to the patch?

Reply:
Done!